docs/status.md

```diff
@@ -131,7 +131,7 @@ one of the following tracking labels.
 | SliceOp |no||||no|
 | SortOp |no||||no|
 | SqrtOp |yes||||no|
-| SubtractOp |no||||no|
+| SubtractOp |yes||||yes|
```

Comment:
Let's also check whether SubtractOp's verifier and shape function conform to the newly written spec. If they do, let's flip `yes*` in the table to `yes`. If they don't let's open tickets about this.